Bottom: de143d0e8db451655b9adc313a215108fad4ff34
Top:    8a94234e75af6a68d14636d0718e4bb082891542
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 15:03:14 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 955322a..7b884d9 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1338,12 +1338,6 @@ static void *statemachine(void *arg)
 		sigwait(&signalset, &sig);
 
 		switch (sig) {
-		case SIGCONT:
-			master_mutex_lock();
-			master_finish(master_list);
-			master_mutex_unlock();
-			break;
-
 		case SIGTERM:
 		case SIGINT:
 		case SIGUSR2:
@@ -1484,7 +1478,7 @@ static void handle_mounts_finish(void)
 	finish_mutex_lock();
 	sdc.busy++;
 	/* Poke signal handler */
-	pthread_kill(state_mach_thid, SIGCONT);
+	pthread_kill(state_mach_thid, SIGTERM);
 	finish_cond_wait();
 	finish_mutex_unlock();
